config/swagger.yaml

```diff
@@ -589,6 +589,64 @@ paths:
           description: Block or account not found
           schema:
             $ref: '#/definitions/Error'
+  /account/txs/{account_pubkey}:
+    get:
+      tags:
+        - internal
+      operationId: GetAccountTranactions
+      description: 'Get accounts''s transactions'
```

Comment:
The description is ambiguous. Does it refer only to txs in longest chain, to txs in mempool, both?

Reply:
Transactions in blocks in the longest chain. Amending the description